dali/kernels/imgproc/convolution/laplacian_cpu_test.cc

```diff
@@ -95,7 +95,7 @@ struct LaplacianWindows {
   static constexpr int window_size = window_size_;
   static constexpr bool use_smoothing = use_smoothing_;
 
-  LaplacianWindows() {
+  LaplacianWindows() : window_sizes{}, windows{} {
```

Comment:
False positive, isn't it?

Reply:
It doesn't make much sense as it is not read anywhere. I'd say that all those issues are the same (not sure about KernelContext ctx; but I can see it is declared all over the tests the same way). On the other hand if there is an error of this kind one day maybe initialization those first makes the error more reproducible at least?